crypto-ffi/Cargo.toml

```diff
@@ -29,7 +29,12 @@ tls_codec.workspace = true
 async-lock.workspace = true
 log.workspace = true
 log-reload.workspace = true
+<<<<<<< HEAD
 serde_json.workspace = true
+=======
+proteus-wasm = { workspace = true, optional = true }
+core-crypto-keystore = { workspace = true, optional = true}
+>>>>>>> 75bea838 (feat: refactor WASM error types)
```

Comment:
Somehow we ended up with conflict markers in this commit.
Could you please fix this?